estimagic/optimization/scipy_optimizers.py

```diff
@@ -0,0 +1,137 @@
+import functools
+
+import numpy as np
+import scipy
+
+from estimagic.optimization.default_algo_options import GRADIENT_TOLERANCE
+from estimagic.optimization.default_algo_options import LIMITED_MEMORY_STORAGE_LENGTH
+from estimagic.optimization.default_algo_options import MAX_CRITERION_EVALUATIONS
+from estimagic.optimization.default_algo_options import MAX_ITERATIONS
+from estimagic.optimization.default_algo_options import MAX_LINE_SEARCH_STEPS
+from estimagic.optimization.default_algo_options import RELATIVE_CRITERION_TOLERANCE
+
+
+DEFAULT_ALGO_INFO = {
```

Comment:
Maybe to config so that everyone can see which entries need to be specified. If it is module specific, I would rename it to ``DEFAULT_SCIPY_ALGO_INFO`` or something similar.

Reply:
I postpone this until we have a bit more experience with other libraries.